include/uichapterstatus.h

```diff
@@ -2,26 +2,26 @@
 #define GUARD_UICHAPTERSTATUS_H
 
 void StartChapterStatusHelpBox(ProcPtr);
-// ??? sub_808DCD0(???);
+// ??? GetFirstValidBlueUnit_Unused(???);
```

Comment:
This doesn't need to happen in this PR, but can we start to expose these functions in these headers?

Reply:
Good idea - added in 67ec833 :)